highlight/highlight
# Add additional error fields to query

## Summary
Support search errors by trace id and secure session id, when using the error_joined_vw.

## How did you test this change?
1) Run migrations.
2) Confirm in clickhouse that the rows are on the table

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
N/A
